pkg/auth/auth.go

```diff
@@ -0,0 +1,11 @@
+// Copyright (C) 2018 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package auth
+
+import (
+	"github.com/zeebo/errs"
+)
+
+// Error default auth error
```

Comment:
The doc comments should sounds like a sentence, e.g.
```suggestion
// Error is the default auth error class
```

Reply:
done